packages/server/lib/server-e2e.ts

```diff
@@ -45,6 +45,8 @@ const isResponseHtml = function (contentType, responseBuffer) {
 
 export class ServerE2E extends ServerBase<SocketE2E> {
   private _urlResolver: Bluebird<Record<string, any>> | null
+  // the initialization of this variable is only precautionary as the actual config value is applied when the server is created
+  private useDefaultDocumentForDomains: string[] | null = null
```

Comment:
Reading through this, I really want this to be something like `excludeDomainInjectionPatterns` or `excludeDomainInjectionList`. I feel like I really only understand `useDefaultDocumentForDomains` because I'm currently reviewing the concept.

Reply:
I also feel like I want this to carry over into the experimental Name. Like `experimentalExcludeDomainInjectionPattern`  or `excludeDocumentDomainList`. I don't even love these names 🤷🏻‍♀️

Reply:
this 💯 . I don't think `experimentalUseDefaultDocumentDomain` is a good name as it isn't entirely representative of what is happening, and it reads like a boolean when the options are a string array or null

Reply:
renamed `experimentalUseDefaultDocumentDomain` and the `useDefaultForDomains` naming to using the `experimentalSkipDomainInjection`nomenclature